popup.js

```
@@ -0,0 +1,441 @@
+    updateEnabledState();
+    loadHistory();
+  } catch (error) {
+    console.error('Error initializing popup:', error);
+  }
+});
+
+/**
+ * Get all patterns from the UI
+ * @returns {Array} Array of pattern objects
+ */
+function getPatterns() {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'getPatterns'. Consider to delete it or rename it.

'getPatterns' is defined here:


(lint/suspicious/noRedeclare)